### PR TITLE
Update mangling prefix in Mangling.rst

### PR DESCRIPTION
**Pull request by adrian-prantl**
I think this is out of date, right?

### CODE REVIEW

File: docs/ABI/Mangling.rst [hunk 1]
```diff
@@ -9,7 +9,7 @@ Mangling
 --------
 ::
 
-  mangled-name ::= '_S' global
+  mangled-name ::= '$' global
```
Comment (eeckstein) [inline docs/ABI/Mangling.rst]: Should be '$S'

**Review by eeckstein — CHANGES_REQUESTED** [review]
Good catch!
The prefix is $S now and will be changed to $s soon.

**Comment by weliveindetail** [comment]
Trying this locally, I get the following - is that expected? Those without leading underscores never finish.
```
$ swift demangle _T04main5PointC6negateyyF
_T04main5PointC6negateyyF ---> main.Point.negate() -> ()
$ swift demangle __$S4main5PointC6negateyyF
_ ---> _
$ swift demangle _$S4main5PointC6negateyyF
_ ---> _
$ swift demangle $S4main5PointC6negateyyF
^C
$ swift demangle __$s4main5PointC6negateyyF
_ ---> _
$ swift demangle _$s4main5PointC6negateyyF
_ ---> _
$ swift demangle $s4main5PointC6negateyyF
^C
```

**Comment by weliveindetail** [comment]
@rintaro Haha, yes.. works now.

**Review by eeckstein — APPROVED** [review]

**Pull request merged**